### ISSUE TITLE
Неквестовые задания

### ISSUE THREAD

**Issue by FaceLess1** [uonline/universe#32]
задания для игроков предусмотрены будут окромя квестов? ну, там чисто поучаствовать в n битвах, заюзать умение...

**Comment by m1kc** [comment]
Это в релиз пойдёт?

Если да, то какие задания будут?

**Comment by FaceLess1** [comment]
ты же раскритиковал мою задумку в ежедневных небольших заданиях.

**Comment by FaNatizM** [comment]
> ты же раскритиковал мою задумку в ежедневных небольших заданиях. 

Ну а что тогда в релиз пойдёт?

**Comment by m1kc** [comment]
> ты же раскритиковал мою задумку в ежедневных небольших заданиях.

Да, такие раскритиковал. Готов согласиться на еженедельные. Но я больше за те, которые можно выполнить один раз.
